user/caching.md

```diff
@@ -134,17 +134,27 @@ caches `$HOME/.cache/pip`.
 
 ### ccache cache
 
-For caching `ccache` files, use:
+If you are using `ccache`, use:
 
 ```yaml
 language: c # or other C/C++ variants
 
 cache: ccache
 ```
 
-caches `$HOME/.ccache`, and adds `/usr/lib/ccache` to the front of `$PATH`.
+to cache `$HOME/.ccache`, and add `/usr/lib/ccache` to the front of `$PATH`.
```

Comment:
Yeah, but don't we already mention that? 

Ah, I see what you mean. Interesting take. I'll reword.